taxonomies/ingredients.txt

```diff
@@ -46472,6 +46671,28 @@ pt:Sumo de vejetais
 ru:Овощной сок
 sv:grönsaksjuice
 
+<en:vegetable
```

Comment:
milk substitute should not be under vegetable, for the ingredients taxonomy, it's a "is a" relationship, "milk substitute" is not a "vegetable", even if it's made from vegetables.

Reply:
Note that the following exists in the taxonomy as well:
```
<en:vegetable fat
en:milk fat substitute
```